### PR TITLE
Update to install.rst

### PR DESCRIPTION
**Pull request by linuxdude21**
First-half installation page change.  I wanted to know what you all think before continuing to update the 2nd half of the installation guide. The biggest changes are the order (flow) of the install guide and OPNsense Importer.

Order:  The page first talks about Image types then jumps to Download & verification (should be closer to the installation section), and then back to installation Media, naming, etc.  The flow in this change is installation images/media, downloading, verifying, and installing.

OPNsense Importer: Complete rewrite to give direction on how to use it.

Removals:
LibreSSL vs OpenSSL
Cut back on the directions for downloading image files, as the previous directions failed after step 2.  Most browsers today download even if you right-click and open in a new tab. (lands signed and verified in the GUI of the running software) Updates:
Installation Media adding Image Type column
add openssl checksum step

### CODE REVIEW

File: source/manual/install.rst [hunk 1]
```diff
@@ -316,14 +303,28 @@ restart the boot procedure.
...
 -----------------
-All images feature the new "opnsense-importer" utility, which is now invoked
-instead of the early installer. You can stop the automatic timeout by pressing
-any key. Afterwards you will have the opportunity to select a disk to import
-from. If the option times out or the importer is exited without a disk selection,
-the factory defaults will be used for the boot.
-
-The next prompt will be for manual interface selection.
-This step is well-established since OPNsense 15.7 .
+All images have the OPNsense Importer feature that offers great flexibility in
```
Comment (fichtner) [inline source/manual/install.rst:306]: Technical docs should avoid opinions and exaggeration, "great" is one of those and can easily be dropped

File: source/manual/install.rst [hunk 2]
```diff
@@ -316,14 +303,28 @@ restart the boot procedure.
...
-instead of the early installer. You can stop the automatic timeout by pressing
-any key. Afterwards you will have the opportunity to select a disk to import
-from. If the option times out or the importer is exited without a disk selection,
-the factory defaults will be used for the boot.
-
-The next prompt will be for manual interface selection.
-This step is well-established since OPNsense 15.7 .
+All images have the OPNsense Importer feature that offers great flexibility in 
+recovering failed firewalls quickly or testing new releases with existing 
+configurations without installing.
```
Comment (fichtner) [inline source/manual/install.rst:308]: closer to the point: without overwriting the current installation by running the new version in memory with the existing configuration

File: source/manual/install.rst [hunk 3]
```diff
@@ -316,14 +303,28 @@ restart the boot procedure.
...
-
-The next prompt will be for manual interface selection.
-This step is well-established since OPNsense 15.7 .
+All images have the OPNsense Importer feature that offers great flexibility in 
+recovering failed firewalls quickly or testing new releases with existing 
+configurations without installing. 
+
+To use OPNsense Importer during the installation boot-up process do the following:
+
+1. You must have a 2nd USB drive formatted with FAT or FAT32 File system.
```
Comment (fichtner) [inline source/manual/install.rst:312]: The early importer can import from USB stick, but also just from the installed disk... (see point above)

File: source/manual/install.rst (hunk 1 above)
Comment (linuxdude21) [reply source/manual/install.rst:306]: I completely agree.  It is an excellent feature if you ever have to recover a bad firewall.

File: source/manual/install.rst [hunk 4]
```diff
@@ -316,14 +303,28 @@ restart the boot procedure.
...
+   a. Preferable non-bootable USB drive.
+2. Create a **conf** directory on the root of the USB drive
+3. Put an unencrypted backup xml or configuration xml into /conf and rename the file to **config.xml**
```
Comment (fichtner) [inline source/manual/install.rst:315]: backup and configuration xml are the same, perhaps shorten to "configuration"

File: source/manual/install.rst [hunk 5]
```diff
@@ -316,14 +303,28 @@ restart the boot procedure.
...
-All images feature the new "opnsense-importer" utility, which is now invoked
-instead of the early installer. You can stop the automatic timeout by pressing
-any key. Afterwards you will have the opportunity to select a disk to import
-from. If the option times out or the importer is exited without a disk selection,
-the factory defaults will be used for the boot.
-
-The next prompt will be for manual interface selection.
-This step is well-established since OPNsense 15.7 .
+All images have the OPNsense Importer feature that offers great flexibility in 
+recovering failed firewalls quickly or testing new releases with existing
```
[the remark this answers is not in the document]
Comment (linuxdude21) [reply source/manual/install.rst:307]: understood.

File: source/manual/install.rst (hunk 2 above)
Comment (linuxdude21) [reply source/manual/install.rst:308]: All images have the OPNsense Importer feature that offers flexibility in recovering failed firewalls or testing new releases without overwriting the current installation by running the new version in memory with the existing configuration.

Better?

File: source/manual/install.rst (hunk 3 above)
Comment (linuxdude21) [reply source/manual/install.rst:312]: Oh, I see, I didn't realize that.  Just to be clear "Installed disk", means the existing drives from a previous installation?

File: source/manual/install.rst (hunk 4 above)
Comment (linuxdude21) [reply source/manual/install.rst:315]: Well really two points here, you can use either a backup XML or a copy /conf/config.xml "configuration".  Another thing is you can encrypt backups, but you can't use encrypted .xml with the Importer.

Comment (fichtner) [reply source/manual/install.rst:315]: yes, that's basically the idea of requiring a /conf directory to be present to import :)

File: source/manual/install.rst (hunk 3 above)
Comment (fichtner) [reply source/manual/install.rst:312]: yes, that's basically the idea of requiring a /conf directory to be present to import :)

File: source/manual/install.rst (hunk 2 above)
Comment (fichtner) [reply source/manual/install.rst:308]: yep

File: source/manual/install.rst (hunk 4 above)
Comment (fichtner) [reply source/manual/install.rst:315]: it's a bit more complex: the backups are just copies of the authoritative config.xml from an older point in time. You can also import backups but these reside in /conf/backups directory and are only used when switching to them via GUI or console. The main config.xml is a configuration XML obtained somehow or edited otherwise and of course unencrypted as the decryption was not implemented for several reasons in this end of the system.

File: source/manual/install.rst [hunk 6]
```diff
@@ -59,105 +59,44 @@ useful for SD memory card installations.
...
-|        | with UEFI support                                 |
-+--------+---------------------------------------------------+
-| nano   | A preinstalled image for >=4 GB USB sticks,       |
-|        | SD or CF cards for use with embedded devices      |
-|        | running in serial console (115200) mode with      |
-|        | secondary VGA support (no kernel messages though) |
-+--------+---------------------------------------------------+
+Depending on your hardware and use case, different installation options are available:
+
++--------+---------------------------------------------------+------------+
```
Comment (fichtner) [inline source/manual/install.rst:68]: nice addition, thanks!

File: source/manual/install.rst [hunk 7]
```diff
@@ -59,105 +59,44 @@ useful for SD memory card installations.
...
+.. Note::
+   All Full Image types can run both **`OPNsense Importer <https://docs.opnsense.org/manual/install.html#opnsense-importer>`** 
+   before booting into the Live environment and also run 
+   **`Installer <https://docs.opnsense.org/manual/install.html#install-to-target-system>`** once booted into the Live environment.
 
 .. Warning::
 
   Flash memory cards will only tolerate a limited number of writes
-  and re-writes. For embedded (nano) versions memory disks for /var and /tmp are
+  and re-writes. For embedded (nano) versions memory disks for **/var** and **/tmp** are
```
Comment (fichtner) [inline source/manual/install.rst:95]: Nowadays this is:

```suggestion
  and re-writes. For embedded (nano) versions memory disks for **/var/log** and **/tmp** are
```

File: source/manual/install.rst [hunk 8]
```diff
@@ -210,23 +149,71 @@ Media Filename Composition
...
+* The openssl public key (<filename>.pub)
+
+Use one of the OPNsense mirrors to download these files:
 
+1. Go to the bottom of OPNSense `download <https://opnsense.org/download>`__ page.
+2. Click one of the available mirrors closest to your location.
+3. Download one of each file mentioned above for your Image type.
 
---------------------
-OpenSSL and LibreSSL
```
Comment (fichtner) [inline source/manual/install.rst:221]: nice catch ;)

**Comment by fichtner** [comment]
Thanks for doing this! I'll probably nag here quite a bit but really appreciate the help. Apologies in advance :)

File: source/manual/install.rst [hunk 9]
```diff
@@ -22,16 +22,16 @@ architecture only.
 Embedded vs Full
 ----------------
 
+Since version 15.1.10 (04 May 2015) the option to install an
```
[the remark this answers is not in the document]
Comment (linuxdude21) [reply source/manual/install.rst:25]: Completely agree with the reference and kinda see your later point.   Merge the embedded/full with installation Media is the question?

File: source/manual/install.rst (hunk 4 above)
Comment (linuxdude21) [reply source/manual/install.rst:315]: Sorry you know the backend of the system more than me.  But from an end user, GUI-based user mostly, I know nothing of the /conf/backups/.  When I talk about backup.xml.  I'm referencing to the exported backups from the System > Configuration > Backups.  "Download configuration" button.  Place that file on a USB drive and use Importer in that context.  Maybe I should organize Importer into two functions, new installation using a backup.xml, and firewall recovery/testing new releases using existing /conf/config.xml on an installed drive?

File: source/manual/install.rst (hunk 9 above)
[the remark this answers is not in the document]
Comment (fichtner) [reply source/manual/install.rst:25]: Merge first and second paragraph, yes. Trouble is the point it is trying to make is weak: embedded exist, full can be installed anywhere -- I don't have a better idea at the moment though.

File: source/manual/install.rst (hunk 4 above)
Comment (fichtner) [reply source/manual/install.rst:315]: well, my initial point about only using "configuration"... perhaps "previously exported configuration" or downloaded... just to be precise about the source and not the type of the configuration (backup vs. main)

Yes, first section for existing installs (shorter) and then second section for preparing a USB stick for import (e.g. new install where you don't have a previous install on the disk you want to retain)

**Comment by linuxdude21** [comment]
No nags at all, I appreciate the insights.  No apologies are needed at all.  I want to know if I'm heading in the right direction.  One bit of help I'm going to need on my side is the proper use of Github.  Do I make these corrections on my local doc and update this pull request?  Honestly, this is the part of Github I just don't understand, forking, push/pulls requests.

**Comment by fichtner** [comment]
The git branch is basically the GitHub PR... you add more changes locally, commit them on top and push to same branch to make the additions appear here.

File: source/manual/install.rst (hunk 9 above)
[the remark this answers is not in the document]
Comment (linuxdude21) [reply source/manual/install.rst:25]: I see your point on those two lines.  However, I didn't do any rewriting as I don't fully understand both lines.  Or should I say the contrast between both lines?  I just reorder the lines to match the Embedded vs Full, and the table below [ Embedded | Full ].   

Isn't the intended purpose of embedded more of the preinstall aspect?  "I'm going to download this image, write it to disk, install that disk into the firewall, and boot (no installation process per se).  Due to the nature of flash memory, do we run embedded versions in RAM drives?

[the remark this answers is not in the document]
Comment (fichtner) [reply source/manual/install.rst:25]: You are right, embedded image is a preinstalled image. That's the main difference. It's tailored to reduce write cycles as well, but can be used anywhere. Maybe that would be a good addition to these paragraphs.

**Comment by linuxdude21** [comment]
Hey folks, Sorry if I'm doing too many PRs.  Please let me know if that is unwanted.  Or I should follow a different process? 

Also, would like to know if I'm going in the right direction.  I don't want to send a bunch of edits that are completely unusable or not accepted.  Thanks Nick,

**Comment by AdSchellevis** [comment]
@linuxdude21 we'll try to discuss next week how we can move this one to a merge, but so far the process from my end looks normal. Just open a new PR (branch) when working on a new topic. When processing feedback, we're used to kep the original one open until it's finished. It has been a busy week, these things sometimes just take time.

**Comment by linuxdude21** [comment]
@AdSchellevis & @fichtner Thank you for the replies.  When I first got into this, I had a very mild idea of what things I wanted to update.  Few section updates, and minor re-ordering.  After that last few nights, it's turning into a complete reordering of the sections.  With most sections needing some form of updating.  I don't mind putting in the time/effort, just don't want it to go to waste because I'm headed in the wrong direction.  Please ignore me I'm sure you have bigger fish to fry.  Have a great weekend!

File: source/manual/install.rst [hunk 10]
```diff
@@ -15,24 +15,36 @@ Architecture
...
+and Full Images.  The Embedded Image is intended for environments where preinstalling 
+the storage media is required due to a lack of local resources on the firewall 
+like storage, and/or console access (VGA/Serial).  The image is tailored to reduce 
+write cycles as well, but the image can be used anywhere.  Another reason for the 
+Embedded Image is to eliminate the need for local console access for installing OPNsense.  
+Installation is managed by prewriting the image to storage, installing the storage, and 
+booting the system.
+
+.. 
+   For nano images is console access required?
```
Comment (AdSchellevis) [inline source/manual/install.rst:35]: it's practical, but when written to a sd card and it boots normally, strictly you don't need it.

File: source/manual/install.rst [hunk 11]
```diff
@@ -210,76 +161,99 @@ Media Filename Composition
...
-  **Please** be aware that the latest installation media does not always
-  correspond with the latest released version. OPNsense installation images are
-  provided on a regular basis together with major versions in January and July.
-  More information on our release schedule is available from our package
-  repository, see `README <https://pkg.opnsense.org/releases/mirror/README>`__
+  **Please** be aware that the latest installation media does not always correspond 
+  with the latest released version available. OPNsense installation images are provided 
+  on a scheduled basis with major release versions in January and July. More information 
+  on our release schedule is available from our package repository, see 
+  `README <https://pkg.opnsense.org/releases/mirror/README>`.  You are encourage to updated
```
Comment (AdSchellevis) [inline source/manual/install.rst:168]: > encourage

encouraged ?

File: source/manual/install.rst [hunk 12]
```diff
@@ -210,76 +161,99 @@ Media Filename Composition
...
+After unpacking the image you can create the installation media. The easiest method to install 
+OPNsense is to use USB "`vga <https://docs.opnsense.org/manual/install.html#installation-media>`_" 
+Image. If your target platform has a serial console interface choose the 
+“`serial <https://docs.opnsense.org/manual/install.html#installation-media>`_” image. If you 
+need to know more about using the serial console interface, consult the :doc:`serial access how-to<how-tos/serial_access>`.
 
-Write the image to a USB flash drive (>=1 GB) or an IDE hard disk,
-either with dd under FreeBSD or under Windows with physdiskwrite
+Write the image to a USB flash drive (>=1 GB) or hard disk, using either dd for Unix-like 
+OSes and for Windows use physdiskwrite or `Etcher <https://www.balena.io/etcher#download-etcher>`_.
```
Comment (AdSchellevis) [inline source/manual/install.rst:246]: quite some people seem to face challenges with `physdiskwrite` thanks for addressing this, maybe we could also add Rufus (https://rufus.ie/nl/) as an option, that seems to be the other popular option.

File: source/manual/install.rst [hunk 13]
```diff
@@ -316,34 +325,88 @@ restart the boot procedure.
...
+users are available, using password `opnsense`. 
+
+.. Note::
+   That the installation media is read-only, which means your current live configuration will 
+   be lost after reboot.
+
+.. 
+   Commect: I suggest we move Nano Image section either after "Install to target system" or to 
+   before "System Boot Preparation".  Also needs a rewrite, I have it 40% done.  Does nano image 
+   use the 2nd interface as a default LAN interface using DHCP on 192.168.1.x network?
```
Comment (AdSchellevis) [inline source/manual/install.rst:396]: The interface configuration should be the same, the build system only "curries" it with memory/console settings (https://github.com/opnsense/tools/blob/fce1fe3fe006dba02be284e86430bf3d2ad9d892/config/23.1/extras.conf#L69-L73) if I'm not mistaken.

Moving the item is probably ok, from the text review it's a bit difficult to envision the final layout in my head.

File: source/manual/install.rst [hunk 14]
```diff
@@ -316,34 +325,88 @@ restart the boot procedure.
...
+
+.. 
+   !rewrite - Using the Nano image for embedded systems, your firewall is already up and running.  
+   The settings to minimize write cycles to relevant partitions by mounting these partitions in 
+   system memory and reporting features are disabled by default.
+.. 
+   Other default settings like interfaces, DHCP, etc.  Or is a console required for nano images?
+.. 
+   "If there are?" What is the required configuration actions to start passing traffic? 
+
```
Comment (AdSchellevis) [inline source/manual/install.rst:409]: see previous comment.

**Review by AdSchellevis — CHANGES_REQUESTED** [review]

**Comment by AdSchellevis** [comment]
> @AdSchellevis & @fichtner Thank you for the replies. When I first got into this, I had a very mild idea of what things I wanted to update. Few section updates, and minor re-ordering. After that last few nights, it's turning into a complete reordering of the sections. With most sections needing some form of updating. I don't mind putting in the time/effort, just don't want it to go to waste because I'm headed in the wrong direction. Please ignore me I'm sure you have bigger fish to fry. Have a great weekend!

@linuxdude21 I've left some small remarks and answers to some of your questions, really love what you're doing with the page. In my humble opinion the sections aren't set in stone, if shifting makes it better readable, we should certainly do so. The only thing that for the whole documentation is more or less fixed is the main menu organization, as that matches most of the product menu layout.  I probably want to do a final review after we pulled it in and build the html, sometimes you notice things in its final shape.

File: source/manual/install.rst (hunk 12 above)
Comment (linuxdude21) [reply source/manual/install.rst:246]: Personally, I've only used phydiskwrite to confirm this install guide.  When using windows, I feel most users would stick to GUI-based tools.  Of course, I think Windows is adopting Linux tools.  Bash cli integration come win12 or 13, I bet.  haha  Should we remove phydiskwrite, and replace that with Etcher and Rufus?  

I will get Rufus added.  Should I get screenshots of both?

File: source/manual/install.rst (hunk 14 above)
Comment (linuxdude21) [reply source/manual/install.rst:409]: Thank you and apology I haven't used the nano image before.  Do we think it's best to list out the default settings for the user to connect to the firewall after booting up?  I'm thinking this would be an extreme use case where the user flashes the image to the drive, installs, and boots without some type of console.

File: source/manual/install.rst (hunk 11 above)
Comment (linuxdude21) [reply source/manual/install.rst:168]: I had this fixed, it reads: We are encouraged to update OPNsense after installation to be on the latest release available

File: source/manual/install.rst [hunk 15]
```diff
@@ -15,24 +15,33 @@ Architecture
...
 
 ----------------
 Embedded vs Full
 ----------------
 
-Full installs can run on `SD memory
-cards <https://en.wikipedia.org/wiki/Secure_Digital>`__, `solid-state
-disks (SSD) <https://en.wikipedia.org/wiki/Solid-state_drive>`__ or
-`hard disk drives
+OPNsense offers two Image Types with all Major releases, Embedded (nano) Image
```
Comment (fichtner) [inline source/manual/install.rst:25]: I'd prefer lowercase when not referencing a trademark or otherwise wanting to highlight it. Serial, VGA, Nano and DVD work for me as image names. Makes it a little easier to process them.

```suggestion
OPNsense offers two Image types with all major releases: embedded and full images. [etc]
```

File: source/manual/install.rst [hunk 16]
```diff
@@ -15,24 +15,33 @@ Architecture
...
+USB disks, `solid-state
+disks (SSD) <https://en.wikipedia.org/wiki/Solid-state_drive>`__, or `hard disk drives
 (HDD) <https://en.wikipedia.org/wiki/Hard_disk_drive>`__.
 
-Since version 15.1.10 (04 May 2015) the option to install an
-`embedded <https://en.wikipedia.org/wiki/Embedded_operating_system>`__
-OPNsense image is also supported.
-
-The main differences between an embedded image and a full image are:
+The main differences between an Embedded (nano) image and a Full images are:
```
Comment (fichtner) [inline source/manual/install.rst:44]: Ideally we shouldn't confuse users with "nano" vs. embedded before the table below. Afterwards, "full" and "embedded" should no longer be references and the real image names should be used instead when required.

Comment (linuxdude21) [reply source/manual/install.rst:44]: So before the table, the image should be referred to as embedded or nano?

File: source/manual/install.rst (hunk 13 above)
Comment (linuxdude21) [reply source/manual/install.rst]: @AdSchellevis thank you for the reply.  But I'm not sure what I'm looking at in that link.  In my PR I have already moved the nano section and rewrote 80% of it.  I will boot up a nano image.  I believe the defaults are like full image defaults, LAN @ 192.168.1.1/24 on the first NIC, with DHCP pool, DNS, & WebGUI enabled. WAN on 2nd NIC w/DHCP IP.  

Thanks,

File: source/manual/install.rst (hunk 16 above)
Comment (linuxdude21) [reply source/manual/install.rst:44]: After rereading your reply, we want to reference embedded before the table. Please let me know if I'm wrong.  I have already update.

File: source/manual/install.rst [hunk 17]
```diff
@@ -15,24 +15,33 @@ Architecture
...
 ----------------
 Embedded vs Full
 ----------------
 
-Full installs can run on `SD memory
-cards <https://en.wikipedia.org/wiki/Secure_Digital>`__, `solid-state
-disks (SSD) <https://en.wikipedia.org/wiki/Solid-state_drive>`__ or
-`hard disk drives
+OPNsense offers two Image types with all major releases: embedded and full images. [etc]
+and Full Images.  The Embedded Image is intended for environments where preinstalling
```
Comment (fichtner) [inline source/manual/install.rst:26]: looks like copy/paste error

```suggestion
OPNsense offers two Image types with all major releases: embedded and full images.
The Embedded Image is intended for environments where preinstalling 
```

File: source/manual/install.rst (hunk 16 above)
Comment (fichtner) [reply source/manual/install.rst:44]: simplify:

```suggestion
The main differences between embedded and full images are:
```

File: source/manual/install.rst [hunk 18]
```diff
@@ -50,117 +59,55 @@ The main differences between an embedded image and a full image are:
 +-----------------------+-----------------------+
 
 
-Embedded images (nano) store logging and cache data in memory only, while full versions
+Embedded (nano) image store logging and cache data in memory only, while Full versions
```
Comment (fichtner) [inline source/manual/install.rst:62]:
```suggestion
Embedded image store logging and cache data in memory only, while full versions
```

File: source/manual/install.rst [hunk 19]
```diff
@@ -50,117 +59,55 @@ The main differences between an embedded image and a full image are:
...
-+--------+---------------------------------------------------+
+Depending on your hardware and use case, different installation options are available:
+
++--------+---------------------------------------------------+------------+
+| Type   | Description                                       | Image Type |
++========+===================================================+============+
+| dvd    | ISO image boots into a live environment in        | Full       |
+|        | VGA-only mode with UEFI support                   |            |
++--------+---------------------------------------------------+------------+
+| vga    | USB image (.img) boots into a live environment    | Full       |
```
Comment (fichtner) [inline source/manual/install.rst:83]:
```suggestion
| vga    | USB image boots into a live environment           | Full       |
```

File: source/manual/install.rst [hunk 20]
```diff
@@ -50,117 +59,55 @@ The main differences between an embedded image and a full image are:
...
+.. Note::
+   All Full Image types can run both **`OPNsense Importer <https://docs.opnsense.org/manual/install.html#opnsense-importer>`** 
+   before booting into the Live environment and also run 
+   **`Installer <https://docs.opnsense.org/manual/install.html#install-to-target-system>`** once booted into the Live environment.
 
-  To enable for non embedded versions: Go to :menuselection:`System --> Settings --> Miscellaneous --> Disk / Memory Settings`,
-  change the setting, then reboot. Consider to enable an external syslog server as well.
+.. Warning::
+   Flash memory cards will only tolerate a limited number of writes and re-writes. For 
+   embedded (nano) versions memory disks for **/var/log** and **/tmp** are applied by
```
Comment (fichtner) [inline source/manual/install.rst:103]:
```suggestion
   Nano image memory disks for **/var/log** and **/tmp** are applied by 
```

File: source/manual/install.rst [hunk 21]
```diff
@@ -50,117 +59,55 @@ The main differences between an embedded image and a full image are:
...
+   **`Installer <https://docs.opnsense.org/manual/install.html#install-to-target-system>`** once booted into the Live environment.
 
-  To enable for non embedded versions: Go to :menuselection:`System --> Settings --> Miscellaneous --> Disk / Memory Settings`,
-  change the setting, then reboot. Consider to enable an external syslog server as well.
+.. Warning::
+   Flash memory cards will only tolerate a limited number of writes and re-writes. For 
+   embedded (nano) versions memory disks for **/var/log** and **/tmp** are applied by 
+   default to prolong CF (flash) card lifetimes.
+   
+   To enable non-embedded versions: Go to :menuselection:`System --> Settings --> Miscellaneous --> Disk / Memory Settings`,
```
Comment (fichtner) [inline source/manual/install.rst:106]:
```suggestion
   To reverse this behaviour go to :menuselection:`System --> Settings --> Miscellaneous --> Disk / Memory Settings`, 
```

File: source/manual/install.rst [hunk 22]
```diff
@@ -209,77 +156,100 @@ Media Filename Composition
...
+
+If the output of the second command is “**Verified OK**”, your image file was verified 
+successfully, and its safe to install from it. Any other outputs, and you may need 
+to check your commands for errors, or the image file may have been compromised.
 
-    Serial connectivity settings for DECXXXX devices can be found  :doc:`here </hardware/serial_connectivity>`
 
 -------------------
-Installation Method
+Installation Media
```
Comment (fichtner) [inline source/manual/install.rst:226]: Is this a typo? We already had "media"?

Comment (fichtner) [reply source/manual/install.rst:226]: Perhaps change this to "Installation media preparation" or similar to match next heading

File: source/manual/install.rst [hunk 23]
```diff
@@ -288,113 +258,213 @@ The device must be the ENTIRE device (in Windows/DOS language: the 'C'
...
 .. rubric:: Install Instructions
    :name: install-to-system
 
-The boot process gives you the opportunity to run several optional configuration
-steps. It has been designed to always boot into a live environment in order to
-be able to access the GUI or even SSH directly. If a timeout was missed simply
-restart the boot procedure.
+OPNsense installation boot process allows us to run several optional configuration steps. The 
+boot process was designed to always boot into the live environment, allowing us to access the 
+GUI or even SSH directly. If a timeout was missed, restart the boot procedure.
```
Comment (fichtner) [inline source/manual/install.rst:319]:
```suggestion
GUI or even SSH directly. If the importer timeout was missed, restart the boot procedure.
```

File: source/manual/install.rst (hunk 21 above)
Comment (linuxdude21) [reply source/manual/install.rst:106]: This was on the original install page.  I believe who wrote was telling everyone you can make full image installations operation like nano images.

File: source/manual/install.rst (hunk 22 above)
Comment (linuxdude21) [reply source/manual/install.rst:226]: No, not in my suggestion to change the sections.  I renamed a few sections.  This was the first one.

File: source/manual/install.rst (hunk 21 above)
Comment (fichtner) [reply source/manual/install.rst:106]: I know but I'm trying to get rid of "embedded" below the image table with the full/embedded column. From this point on "full" and "embedded" should be avoided.

Comment (linuxdude21) [reply source/manual/install.rst:106]: "To enable full versions" ?

Comment (linuxdude21) [reply source/manual/install.rst:106]: When you are referencing two different items.  In this case, Nano vs Full images.  We should capitalize the first letter.  I will take your lead.

Comment (fichtner) [reply source/manual/install.rst:106]: well, yes but without either "embedded" or "full".

Comment (fichtner) [reply source/manual/install.rst:106]: capitalizing breaks the reading flow for me. For Nano, VGA, Serial and DVD it makes sense, for everything else not.

**Comment by linuxdude21** [comment]
@fichtner Thank you for your edits.  Not a problem. Post-merge is the only way to see these suggested changes.  The most significant change is the flow of the doc.  I have not finished the last Section, "Initial Configuration".  As I wanted to focus more on the installation side of things.  One my steps was to get a nano image up and running and fell into a rabbit whole with NAT.


Unrelated to the install doc:
Where I found a significant flaw in NATing.  Not sure if this is a bug or not: [my post on the issue](https://forum.opnsense.org/index.php?topic=32760.0) But doing a simple Port Forward with an associated filter does not work.

File: source/manual/install.rst (hunk 21 above)
Comment (linuxdude21) [reply source/manual/install.rst:106]: Again, I'll take your lead on this.  But breaking the flow of the reader is precisely the effect people need.  To know there is a difference between the two.  Remember, good man, you know this stuff WELL.  People reading this install guide have no clue.  They are still wrapping their minds around OPNsense.  ;)

File: source/manual/install.rst (hunk 22 above)
Comment (linuxdude21) [reply source/manual/install.rst:226]: > Perhaps change this to "Installation media preparation" or similar to match next heading

Download and Verification are also parts of media preparation, which comes before Installation Media.  So I don't like that it.   But will take your lead if that's the case.

File: source/manual/install.rst (hunk 13 above)
Comment (linuxdude21) [reply source/manual/install.rst]: So I have already moved the section and will boot up a nano image to see the defaults.  I will make future suggestions or have questions

File: source/manual/install.rst (hunk 12 above)
Comment (linuxdude21) [reply source/manual/install.rst]: @AdSchellevis remove physdiskwrite? Again I've never used it before myself.  Windows users are going to have much more success with Etcher and Rufus.

**Comment by AdSchellevis** [comment]
@linuxdude21 thanks a lot for the updates, made some small changes in https://github.com/opnsense/docs/commit/349fed697443d82177f29cdf57e75644ea9debad (links and added an index due to the length of the page). Results are already published online https://docs.opnsense.org/manual/install.html

About your forum question, I see there's communication there, one tool that's always very helpful when debugging issues is the packet capture in interface/diagnostics (apart from the live log you're already using).

**Comment by linuxdude21** [comment]
@AdSchellevis Wow, you already merged.  HAHA this is great.  Several items were missed. I believe that's my fault, as I started using comments within the doc.  With all my edits, they were easy to miss.  I submit another PR.  I finally got the upstream master to update my local repo.  Still trying to under github/repos.  

Thank you for the packet capture suggestion. I had already used it on this problem.  I saw the firewall sending the packet out the WAN interface, but my client never got them.  Franco got the problem disable reply-to on WAN, but I still question why that setting is enabled by default.  Another topic for a different day.  Thanks again, guys.

**Comment by AdSchellevis** [comment]
@linuxdude21 ok, let's try to finish this as soon as possible then. 

The default reply-to//route-to is mostly for historic reasons, when using multiwan setups you do need to know where to return packets to as default routing doesn't apply. These rules ar a bit wider than you might want, but without adding a lot of hard to follow complexity not really solvable. When not using multi-wan, it's always safe to disable both, when you do need multiwan and "on net" traffic, you just need additional rules to make that explicit (disabling the reply-to).
